cosmos-gui/src/cmd_runner.js

```diff
@@ -28,18 +28,19 @@ var spawn = require('child_process').spawn;
 
 function run(cmd, params, callback) {
     var execution = spawn(cmd, params);
-    var result = '';
+    var resStdout = '';
+    var resStderr = '';
 
     execution.stdout.on('data', function (data) {
-        result += 'stdout: ' + data.toString();
+        resStdout += data.toString();
     });
 
     execution.stderr.on('data', function (data) {
-        result += 'stderr: ' + data.toString();
+        resStderr += + data.toString();
     });
 
     execution.on('close', function (code) {
-        return callback(null, result);
+        return callback(null, resStdout.length == 0 ? null : resStdout);
```

Comment:
`NaN` appeared because of a wrong string concatenation, thanks @gtorodelvalle for looking!

Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/95ccdc4dc1dbba0c8907a822ebf7f9f6d2fddfc5